eog/eogtricks-quickmove.plugin

```diff
@@ -0,0 +1,9 @@
+[Plugin]
+Loader=python3
+Module=eogtricks-quickmove
+IAge=3
+Name=[EOGtricks] Quick move to folder
+Icon=folder-new
+Description=Press N to select target folder, M to move current image there
+Authors=Andrew Chadwick <a.t.chadwick@gmail.com>, Florian 'floe' Echtler <floe@butterbrot.org>
+Copyright=Copyright © 2017 Andrew Chadwick <a.t.chadwick@gmail.com>
```

Comment:
It's reasonable to bump the date here, I guess. Only suggesting because I'm bound to forget...